src/components/organisms/NewProfileModal/ProfileModal/ProfileModalContent/index.ts

```diff
@@ -0,0 +1 @@
+export { ProfileModalContent } from "components/organisms/NewProfileModal/ProfileModal/ProfileModalContent/ProfileModalContent";
```

Comment:
since it's index file, just export it from `./ProfileModalContent `
```suggestion
export { ProfileModalContent } from "./ProfileModalContent";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/966a98a7c5783a85a2cf8361a69adb3bc783f26b